plugins/06_WarlordsOfDraenor/zones/frostfire.lua

```diff
@@ -37,13 +51,823 @@ map.nodes[64805300] = Gorok
+}) -- Gruuk
+
+map.nodes[54602220] = Rare({
+    id = 71665,
+    quest = 32918,
+    rewards = {Item({item = 111530})}
+}) -- Giant-Slayer Kul
+
+map.nodes[54606940] = Rare({
+    id = 76914,
+    quest = 34131,
+    rewards = {Item({item = 111484})}
+}) -- Coldtusk
+
+map.nodes[58603420] = Rare({id = 78151, quest = 34130}) -- Huntmaster Kuang
+
+map.nodes[61602640] = Rare({
+    id = 79678,
+    quest = 34708,
+    rewards = {Item({item = 112078})}
+}) -- Jehil the Climber
+
+map.nodes[66403140] = Rare({
+    id = 74613,
+    quest = 33843,
+    rewards = {Item({item = 111533})}
+}) -- Broodmother Reeg'ak
+
+map.nodes[67407820] = Rare({
+    id = 78621,
+    quest = 34477,
+    rewards = {Item({item = 112086})}
+}) -- Cyclonic Fury
+
+map.nodes[68801940] = Rare({
+    id = 87348,
+    quest = 37382,
+    rewards = {Item({item = 119415})}
+}) -- Hoarfrost
+
+map.nodes[70002700] = Rare({
+    id = 87351,
+    quest = 37381,
+    rewards = {Item({item = 119376})}
+}) -- Mother of Goren
+
+map.nodes[70003600] = Rare({
+    id = 72364,
+    quest = 33562,
+    rewards = {Item({item = 111545})}
+}) -- Gorg'ak the Lava Guzzler
+
+map.nodes[70603900] = Rare({
+    id = 87356,
+    quest = 37379,
+    rewards = {Item({item = 119416})}
+}) -- Vrok the Ancient
+
+map.nodes[71404680] = Rare({
+    id = 74971,
+    quest = 33504,
+    rewards = {Item({item = 107661})}
+}) -- Firefury Giant
+
+map.nodes[72203300] = Rare({
+    id = 78265,
+    quest = 34361,
+    rewards = {Item({item = 111534})}
+}) -- The Bone Crawler
+
+map.nodes[72203600] = Rare({
+    id = 87352,
+    quest = 37380,
+    note = L['gibblette_the_cowardly_note'],
+    rewards = {Item({item = 119349}), Toy({item = 119180})}
+}) -- Gibblette the Cowardly
+
+map.nodes[72402420] = Rare({
+    id = 87357,
+    quest = 37378,
+    rewards = {Item({item = 119416})}
+}) -- Valkor
+
+map.nodes[76406340] = Rare({
+    id = 77526,
+    quest = 34132,
+    rewards = {Item({item = 112094})}
+}) -- Scout Goreseeker
+
+map.nodes[83604720] = Rare({
+    id = 87622,
+    quest = 37402,
+    rewards = {Item({item = 119366})}
+}) -- Ogom the Mangler
+
+map.nodes[85005220] = Rare({id = 87600, quest = 37556}) -- Jaluk the Pacifist
+
+map.nodes[86604880] = Rare({
+    id = 84392,
+    quest = 37401,
+    rewards = {Item({item = 119359})}
+}) -- Ragore Driftstalker
+
+map.nodes[86605180] = Rare({
+    id = 84376,
+    quest = 37403,
+    rewards = {Item({item = 119374})}
+}) -- Earthshaker Holar
+
+map.nodes[87004640] = Rare({
+    id = 84374,
+    quest = 37404,
+    rewards = {Item({item = 119372})}
+}) -- Kaga the Ironbender
+
+map.nodes[88605740] = Rare({
+    id = 84378,
+    quest = 37525,
+    rewards = {Item({item = 119365})}
+}) -- Ak'ox the Slaughterer
+
 map.nodes[13205060] = Rare({
     id = 81001,
     rewards = {
         Mount({item = 116794, id = 657}) -- Garn Nighthowl
     }
 }) -- Nok-Karosh
 
+map.nodes[62604220] = Rare({
+    id = 72156,
+    note = L['borrok_the_devourer_note'],
```

Comment:
This phrase doesn't exist in the translation files.

Reply:
Pushed an update for this